public/sass/components/_dropdown.scss

```diff
@@ -260,19 +260,17 @@
   margin-bottom: -2px;
   @include border-radius(5px 5px 5px 0);
 }
-
-// Caret to indicate there is a submenu
-.dropdown-submenu > a::after {
-  position: absolute;
-  top: 35%;
-  right: $space-sm;
-  background-color: transparent;
-  color: $text-color-weak;
-  font: normal normal normal $font-size-sm/1 FontAwesome;
-  content: '\f0da';
-  pointer-events: none;
-  font-size: 11px;
-}
+// .dropdown-submenu > a::after {
```

Comment:
I have commented out this className as I've seen it used in one more component. So after the migration of that component, I will delete it.